omniauth-greenhouse.gemspec

```diff
@@ -20,6 +20,6 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency 'bundler', '~> 1.3'
   spec.add_development_dependency 'rake'
 
-  spec.add_runtime_dependency 'omniauth-oauth2', '~> 1.1.1'
-  spec.add_runtime_dependency 'omniauth', '~> 1.2.1'
+  spec.add_runtime_dependency 'omniauth-oauth2', '~> 1'
+  spec.add_runtime_dependency 'omniauth', '>= 1.3.1', '< 2'
```

Comment:
Do we need this line anymore?  I'm pretty sure it's already a dependency of `omniauth-oauth2`